CMakeTests/FindALSA.cmake

```diff
@@ -39,14 +39,19 @@ if(ALSA_INCLUDE_DIR AND EXISTS "${ALSA_INCLUDE_DIR}/alsa/version.h")
   unset(alsa_version_str)
 endif()
 
-include(${CMAKE_CURRENT_LIST_DIR}/FindPackageHandleStandardArgs.cmake)
+include(FindPackageHandleStandardArgs)
 FIND_PACKAGE_HANDLE_STANDARD_ARGS(ALSA
                                   REQUIRED_VARS ALSA_LIBRARY ALSA_INCLUDE_DIR
                                   VERSION_VAR ALSA_VERSION_STRING)
 
 if(ALSA_FOUND)
   set( ALSA_LIBRARIES ${ALSA_LIBRARY} )
   set( ALSA_INCLUDE_DIRS ${ALSA_INCLUDE_DIR} )
+  add_library(ALSA::ALSA UNKNOWN IMPORTED)
+  set_target_properties(ALSA::ALSA PROPERTIES
+    IMPORTED_LOCATION ${ALSA_LIBRARY}
+    INTERFACE_INCLUDE_DIRECTORIES ${ALSA_INCLUDE_DIR}
```

Comment:
Precisely, that's why I always insist on using imported targets that are always supposed to be correct instead of some random variables created by some script.
The day a new required variable that is added (like extra cflags), you fail to update your code and it doesn't work anymore...